documentation/src/main/asciidoc/mapping-streams.asciidoc

```diff
@@ -0,0 +1,67 @@
+[[mapping-streams]]
+== Mapping Streams
+
+The mapping of `java.util.Stream` is done in a similar way as the mapping of collection types, i.e. by defining mapping
+methods with the required source and target types in a mapper interface.
+
+The generated code will contain the creation of a `Stream` from the provided `Iterable`/`Array` or will collect the
+provided `Stream` into an `Iterable`/`Array`. If a mapping method or an implicit conversion for the source and target
+element types exists, then this conversion will be done in `Stream.map`. The following shows an example:
```

Comment:
done via `Stream#map()` (let's use the hash pound when referencing to a method).